### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by remydev**
Simple install information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,42 @@
...
+psql -d <database>
+```
 ```sql
 create table users(id int, created_at date, updated_at date, username text, realname text, password text, avatar text, perms text[]);
 ```
 
+## start
+
+```shell
+git clone <branch>
```
Comment (Lukc) [inline README.md]: Should actually be the URL in this command, not the branch.

Those are instructions to clone a git repository and build its content. I’m not sure the cloning part is necessary to include, but I’m not against it. Most of the following instructions are about building it though, so maybe rename the section from “start” to “Build & Run” or something similar?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,42 @@
...
+- shards
+- postgresql
+
+Usually install on Aarch like
+```shell
+sudo pacman -S crystal shards git vim postgresql
+```
+## Database init
+```shell
+sudo su -l postgres
```
[the remark this answers is not in the document]
Comment (remydev) [reply README.md]: this is for someone like me.
I wish keep it

[the remark this answers is not in the document]
Comment (blankoworld) [reply README.md]: Either you add "#" before. Or you keep it as this to simplify user's use.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,42 @@
...
+initdb -D '/var/lib/postgres/data'
+sudo systemctl start postgresql.service
+createuser --interactive 
+#create <dbuser>
+createdb <database>
```
[the remark this answers is not in the document]
Comment (blankoworld) [reply README.md]: I usually do this:

```bash
sudo su - postgres
```

And then I follow this: https://olivier.dossmann.net/wiki/services/postgres/#cr%C3%A9er-une-nouvelle-base-de-donn%C3%A9es-avec-un-nouvel-utilisateur

```bash
createuser od
createdb -O od -E UTF-8 mydb
psql mydb
```

and:

```sql
ALTER USER olivier WITH PASSWORD 'mot2passe';
\q
```

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (remydev) [reply README.md]: so i keep ot

**Comment by remydev** [comment]
And now ?

**Pull request closed without merging**